Tasks/Src/Task_CANBusProducer.c

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
+#include "Tasks.h"
+#include "BSP_CAN.h"
+#include "CANbus.h"
+#include "CAN_Queue.h"
+
+void Task_CANBusProducer(void *p_arg) {
+    (void)p_arg;
+
+    CANbus_Init((bool) p_arg, false);
+    
+    while(1) {
+      //call CANBUS_read and receive data in id/buffer, store into a 
```

Comment:
put these comments in the function header cuzzo

Reply:
nuh uh (yeah u right)